tests/core/input/test_input_event_key.h

```diff
@@ -125,6 +125,12 @@ TEST_CASE("[InputEventKey] Key correctly converts itself to text") {
 	none_key.set_physical_keycode(Key::ENTER);
 	CHECK(none_key.as_text() == "Ctrl+Enter (Physical)");
 
+	// Key is None WITH a physical key AND multiple modifiers, checks for correct ordering.
+	none_key.set_alt_pressed(true);
+	none_key.set_shift_pressed(true);
+	CHECK(none_key.as_text() != "Ctrl+Shift+Alt+Enter (Physical)");
+	CHECK(none_key.as_text() == "Ctrl+Alt+Shift+Enter (Physical)");
```

Comment:
This will fail on macOS, since `Alt` is called `Option`:

```
Ctrl+Option+Shift+Enter (Physical)
```

So it should have `ifdef` to account for this or construct test string using `keycode_get_name_by_index` to construct test name.